projects/js-packages/charts/src/components/line-chart/stories/index.stories.tsx

```diff
@@ -359,78 +271,6 @@ BrokenLine.parameters = {
 	},
 };
 
-export const WithStartGlyphs: StoryObj< typeof LineChart > = Template.bind( {} );
-WithStartGlyphs.args = {
-	...Default.args,
-	withStartGlyphs: true,
-};
-
-export const WithCustomGlyph: StoryObj< typeof LineChart > = Template.bind( {} );
-WithCustomGlyph.args = {
-	...Default.args,
-	showLegend: true,
-	withStartGlyphs: true,
-	withLegendGlyph: true,
-	renderGlyph: ( { color, size, x, y } ) => {
-		return <GlyphStar top={ y } left={ x } size={ size * size } fill={ color } />;
-	},
-	glyphStyle: {
-		radius: 10,
-	},
-};
-
-const CustomStarGlyph = ( { color, size, x, y } ) => {
-	const hasXY = typeof x === 'number' && typeof y === 'number' && ( x !== 0 || y !== 0 );
-	const groupProps = hasXY ? { transform: `translate(${ x }, ${ y })` } : {};
-	return (
-		<g { ...groupProps }>
-			<svg
-				xmlns="http://www.w3.org/2000/svg"
-				width={ size * 2 }
-				height={ size * 2 }
-				viewBox="0 0 24 24"
-				style={ { overflow: 'visible', pointerEvents: 'none' } }
-			>
-				<path
-					d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"
-					fill={ color }
-					stroke={ color }
-					strokeWidth="2"
-					strokeLinecap="round"
-					strokeLinejoin="round"
-					transform="translate(-12, -12)"
-				/>
-			</svg>
-		</g>
-	);
-};
-
-export const WithCustomSvgGlyph: StoryObj< typeof LineChart > = Template.bind( {} );
-WithCustomSvgGlyph.args = {
-	...Default.args,
-	showLegend: true,
-	withStartGlyphs: true,
-	withLegendGlyph: true,
-	renderGlyph: ( { color, size, x, y } ) => (
-		<CustomStarGlyph color={ color } size={ size } x={ x } y={ y } />
-	),
-	glyphStyle: {
-		radius: 8,
-	},
-};
```

Comment:
These were all moved to glyph.stories.tsx in https://github.com/Automattic/jetpack/pull/43922 but were reinstated by https://github.com/Automattic/jetpack/pull/43875, I assume due to a conflict.